packages/images/package.json

```diff
@@ -21,12 +25,14 @@
   },
   "homepage": "https://github.com/appium/appium-plugins#readme",
   "dependencies": {
-    "@appium/base-plugin": "^1.5.0"
+    "@appium/base-plugin": "^1.5.0",
```

Comment:
Not for this PR, but just I thought:

it was probably good to print author, homepage and license with `--details` option etc as part of the below. We'll also can guide reporters to check `--details` option to ensure where they can report issues. (or only homepage??)

e.g.
```
kazu$ appium driver list --details
✔ Listing available drivers
- xcuitest@3.31.5 [installed (NPM)], Appium <maintainers@appium.io>,  https://github.com/appium/appium-plugins#readme, Apache-2.0
kazu$ appium plugin list
✔ Listing available plugins
```

Reply:
yes, that's a nice feature idea!